src/lib.rs

```diff
@@ -32,7 +33,7 @@ impl AssetLoader for StlLoader {
         &'a self,
         reader: &'a mut Reader,
         _settings: &'a (),
-        _load_context: &'a mut LoadContext,
+        load_context: &'a mut LoadContext,
```

Comment:
Sorry, this is a bug fix for the wireframe feature. Seems like i made this mistake while upgrading to bevy 0.12 https://github.com/nilclass/bevy_stl/pull/12 . Let me know if you'd prefer a separate commit/PR for this.